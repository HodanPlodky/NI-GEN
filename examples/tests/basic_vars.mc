int inc(int n) {
    return n + 1;
}

int main() {
    int x;
    x = 5;
    return inc(x) + 2;
}
