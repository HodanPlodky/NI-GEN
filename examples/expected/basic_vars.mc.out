input file : examples/tests/basic_vars.mc
return 8
